tests/acceptance/pre-suite/02_chocolatey_application_install.rb

```diff
@@ -0,0 +1,48 @@
+require 'master_manipulator'
+require 'chocolatey_helper'
+test_name 'MODULES-3043 - C97739 - Install Client on Virgin System'
+
+chocolatey_pp = <<MANIFEST
+  class {'chocolatey':
+    chocolatey_download_url => 'file:///C:/chocolatey.nupkg',
+    use_7zip                => false,
+  }
+MANIFEST
+
+chocoVersion = /[0-9]+[\d'.']*/
+
+# Setup
+step 'Inject "site.pp" on Master'
+site_pp = create_site_pp(master, :manifest => chocolatey_pp)
+inject_site_pp(master, get_site_pp_path(master), site_pp)
+
+#Test
+confine_block(:to, :platform => 'windows') do
+
+  agents.each do |agent|
+    opts = {
+        :acceptable_exit_codes => [0, 2]
+    }
+
+    url = get_latest_chocholatey_download_url
+
+
+    step 'Download chocolatey nuget package' do
+      on(agent, "curl #{url} > C:/chocolatey.nupkg")
```

Comment:
You should use the Beaker built-in method [curl_on](http://www.rubydoc.info/github/puppetlabs/beaker/Beaker/DSL/Helpers/HostHelpers:curl_on).